packages/google-cloud/README.md

```diff
@@ -0,0 +1,1123 @@
+
+#### Using the Cloud Storage API module
+
+```
+$ npm install --save @google-cloud/storage
+```
+
+```js
+var storage = require('@google-cloud/storage');
+```
+
+#### Preview
+
+```js
+var fs = require('fs');
+
+// Authenticating on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authentication section above).
+
+var gcs = storage({
+  projectId: 'grape-spaceship-123',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Create a new bucket.
+gcs.createBucket('my-new-bucket', function(err, bucket) {
+  if (!err) {
+    // "my-new-bucket" was successfully created.
+  }
+});
+
+// Reference an existing bucket.
+var bucket = gcs.bucket('my-existing-bucket');
+
+// Upload a local file to a new file to be created in your bucket.
+bucket.upload('/photos/zoo/zebra.jpg', function(err, file) {
+  if (!err) {
+    // "zebra.jpg" is now in your bucket.
+  }
+});
+
+// Download a file from your bucket.
+bucket.file('giraffe.jpg').download({
+  destination: '/photos/zoo/giraffe.jpg'
+}, function(err) {});
+
+// Streams are also supported for reading and writing files.
+var remoteReadStream = bucket.file('giraffe.jpg').createReadStream();
+var localWriteStream = fs.createWriteStream('/photos/zoo/giraffe.jpg');
+remoteReadStream.pipe(localWriteStream);
+
+var localReadStream = fs.createReadStream('/photos/zoo/zebra.jpg');
+var remoteWriteStream = bucket.file('zebra.jpg').createWriteStream();
+localReadStream.pipe(remoteWriteStream);
+```
+
+
+## Google Compute Engine
+
+- [API Documentation][gcloud-compute-docs]
+- [Official Documentation][cloud-compute-docs]
+
+#### Using the all-in-one module
+
+```
+$ npm install --save google-cloud
+```
+
+```js
+var gcloud = require('google-cloud');
+var compute = gcloud.compute;
+```
+
+#### Using the Compute Engine API module
+
+```
+$ npm install --save @google-cloud/compute
+```
+
+```js
+var compute = require('@google-cloud/compute');
+```
+
+#### Preview
+
+```js
+// Authenticating on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authentication section above).
+
+var gce = compute({
+  projectId: 'grape-spaceship-123',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Create a new VM using the latest OS image of your choice.
+var zone = gce.zone('us-central1-a');
+var name = 'ubuntu-http';
+
+zone.createVM(name, { os: 'ubuntu' }, function(err, vm, operation) {
+  // `operation` lets you check the status of long-running tasks.
+
+  operation
+    .on('error', function(err) {})
+    .on('running', function(metadata) {})
+    .on('complete', function(metadata) {
+      // Virtual machine created!
+    });
+});
+```
+
+## Google Prediction API
+
+- [API Documentation][gcloud-prediction-docs]
+- [Official Documentation][cloud-prediction-docs]
+
+#### Using the all-in-one module
+
+```
+$ npm install --save google-cloud
+```
+
+```js
+var gcloud = require('google-cloud');
+var prediction = gcloud.prediction;
+```
+
+#### Using the Prediction API module
```

Comment:
Should we be documenting both the all-in-one and the standalone here? I feel like if users are reading this, there's a good chance they're using the umbrella package. It might be nice to have a link to the standalone package on NPM or GitHub though.